lib/collection/response.js

```diff
@@ -102,6 +102,132 @@ var util = require('../util'),
      */
     supportsBuffer = (typeof Buffer !== undefined) && _.isFunction(Buffer.byteLength),
 
+    /**
+     * @private
+     * Regex to match filename and it's value in content-disposition header
+     */
+    filenameMatchRegex = /filename[^;\n=]*=(['"])?(([^;\n]*)[.]([^.;\n]*)|[^.;\n]*)\1/,
+
+    /**
+     * @private
+     * Regex to match filename* and it's value in content-disposition header
+     */
+    fileEncodingType = /filename\*=[^;\n=]*?"?([^"']*)''(.*)(")?/,
```

Comment:
missing handling language chars within two single quotes

Reply:
Need to figure out
 - Is the encoding optional? If yes is there a default value?

Reply:
add regex to the name